Components/CreateTrip/CreateTrip.js

```diff
@@ -1,38 +1,145 @@
 import React from 'react';
-import { StyleSheet, Text, View, TextInput, TouchableOpacity } from 'react-native';
+import { Image, StyleSheet, Text, View, TextInput, TouchableOpacity } from 'react-native';
 
-export const CreateTrip = () => {
+export const CreateTrip = ({ route }) => {
```

Comment:
https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/checking-out-pull-requests-locally